openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -93,7 +94,7 @@ def _verify_full_shareable_account_response(self, response):
         self.assertIsNone(data["profile_image"])
         self.assertIsNone(data["time_zone"])
         self.assertIsNone(data["languages"])
-        self.assertIsNone(data["bio"])
+        self.assertEqual("Tired mother of twins", data["bio"])
```

Comment:
Actually, I already had test coverage for that at the end of test_patch_account:
````
# If there are no values that would fail validation, then empty string should be supported.
self.send_patch(self.client, {field: ""})

get_response = self.send_get(self.client)
self.assertEqual("", get_response.data[field])